src/components/Lenkepanel/LenkepanelWrapper.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Element, Undertekst, Undertittel } from 'nav-frontend-typografi';
+import { LenkepanelBase } from 'nav-frontend-lenkepanel';
+
+import './LenkepanelWrapper.less';
+import { Periode, Arbeidsgiver, Sykmelding } from '../../types/sykmeldingTypes';
+import dayjs from 'dayjs';
+
+interface LenkepanelProps {
+    lenke: string;
+    tittel: string;
+    sykmeldinger: Sykmelding | Sykmelding[];
+    tekst: string;
+    svg: string;
+}
+
+const beregnPeriodelengde = (fom: Date, tom: Date): number => dayjs(tom).diff(dayjs(fom), 'day');
```

Comment:
Samme som under. Gir dette forresten "riktig" beregning? Jeg har laget en funksjon (med tester) i sykmeldinger-appen som gjør samme beregning, men den legger til 1 eller 2 dager for å inkludere start og slutt-dato. Bør kanskje trekke inn den.

Reply:
da tar jeg i bruk den som allerede finnes :)